manifest.config.ts

```diff
@@ -31,6 +31,12 @@ export default defineManifest(async (env) => ({
       matches: ['*://*/*'],
       run_at: 'document_end',
     },
+    {
```

Comment:
Add as many scripts you want here, `vite` will automatically pick and resolve these